.travis.yml

```diff
@@ -27,89 +27,81 @@ stages:
 jobs:
   fast_finish: true
   include:
-    -
+    - bundler_args: --with system_tests
```

Comment:
I thought `--with` as a parameter was deprecated. I've started to set it as an env var. That way you don't have to repeat it either:
```yaml
env:
  BUNDLE_WITH: system_tests
```

Reply:
neat

Reply:
Created https://github.com/puppetlabs/pdk-templates/pull/381 to implement that across all modules. Only the duplication doesn't go away, because we have cells both ways. since it is through pdk-templates :shrug: it's not a big deal, and the environment var is nice because it is really "global" for all bundler commands in that run.